src/kedro_templar/process_templates.py

```diff
@@ -99,30 +40,24 @@ def download_file_from_s3(file_name, bucket):
     default=False,
     type=bool
 )
-@click.option(
-    "-u",
-    "--upload",
-    "upload_source",
-    required=False,
-    default=False,
-    type=bool
-)
 def process_templates(
     config_path: Text,
     templates_dir: Text,
     output_dir: Text,
     replace_existing: bool,
     upload_source: Text
 ):
-
-    config = load_config( s3_config if s3_config else config_path)
-    environment = create_environment(templates_dir)
+    # extend -c to accept also s3 path, @WOJTEK what do you think?
```

Comment:
As mentioned in other command, we want to have 3 separate commands. See jira task's comment section.

Reply:
Also, use github conversation system for communication instead of comments

Reply:
ok, thanks! changed implementetion